draft-ietf-rats-architecture.md

```diff
@@ -998,11 +998,17 @@ an Attestation Result for any other purpose.
 
 Any solution that conveys information used for security purposes, whether
 such information is in the form of Evidence, Attestation Results,
-Endorsements, or Appraisal Policy, needs to support end-to-end integrity protection
+Endorsements, or Appraisal Policy must support end-to-end integrity protection
 and replay attack prevention, and often also needs to support additional
-security protections.  For example, additional means of authentication,
-confidentiality, integrity, replay, denial of service and privacy
-protection are needed in many use cases.
```

Comment:
The "example" restates the list of 'must' be supported security properties. This seems redundant and/or isn't a real example. Might be better to say "must support end-to-end integrity and replay attack prevention, and often needs to support authentication, confidentiality, denial-of-service and privacy protection."